pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -481,6 +496,9 @@ func (pp *predicatesPlugin) OnSessionOpen(ssn *framework.Session) {
 				status := podAffinityFilter.Filter(context.TODO(), state, task.Pod, nodeInfo)
 				podAffinityStatus := api.ConvertPredicateStatus(status)
 				if podAffinityStatus.Code != api.Success {
+					// TODO: Currently, preemption is not supported when Pod affinity filtering fails.
```

Comment:
It is recommended that you register an issue that describes the todo and update the issue link to the todo comment.